glean-core/src/util.rs

```diff
@@ -121,4 +183,19 @@ mod test {
             fixed_now
         );
     }
+
+    #[test]
```

Comment:
Can we get one test on an ASCII value, just to make sure? e.g. `"1234567890abcdef"` truncated to 10.